community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
++++
+
+toc=true
+
++++
+
+# [Name Option 1] Workplace Harassment - Code of Conduct 
+# [Name Option 2] An informal guide to avoid and deal with events of harassment and bullying
+# [Name Option 3] Workplace Harassment - An informal guide
+
+## Some basic fundamentals
+
+1. Humans are flawed and make mistakes on a regular basis while discovering new ways to hurt new people.
+
+2. We don't live in an ideal world, so we are not dealing with ideal scenarios.
+
+3. But the above statement doesn't provide wind to the practicality argument. Practicality argument is an exceptionally dangerous proposition in itself. So any time any particular statement is backed up by "being practical", it should take care of few things :-
+   1. Being practical means what can be done or is possible in some finite duration. So it should be aggressively progressive in nature and it shouldn't settle on "n" but "n + ∞". We can start from "n" but the vision should be clear to reach "n + ∞" as fast as possible.
+   2. It should be “enabler” in nature.
+
+4. Any democratic argument in case of harassment should be thrown out of the window immediately. For example, "9 out of 10 people are okay with this".
+
+5. Humans are not rational, so don't expect them to behave rationally. We are totally full of feelings.
+
+6. We don't accept snowflake argument. Some people have thick skin and are borderline stoic, and they think that "I am responsible for my feelings" and on the opposite end of emotional spectrum, are people who think that "You don't get to decide if you hurt someone". We will try to have an inclusive space for both of these people.
+
+7. Humans hate to forgive people if forgiving is tough.
+
+8. Convincing a human that they are wrong or have done wrong is exceptionally tough.
+
+9. Humans have double standards. Deal with it.
```

Comment:
Again, this line sounds needlessly disparaging.

Reply:
And is in stark contrast to the point made in 6.

Reply:
Why this line is there :- 
1. It might happen that in the past due to some reason a group of people dealt with a situation in a way different than they are going to deal with it now, which they say might find more appropriate with regards to the current scenario, so that group shouldn't be bound by their past decisions, "you guys did x in scenario y, but now you guys are doing x1 in scenario y" , you can't have double standards and all that. Anyone shouldn't be bound by their past decisions. In some cases there might be more aggressive response due to general affection of the team towards the person, and in some case more passive. 
2. >And is in stark contrast to the point made in 6.
Not clear to me how ? Just because we want a harmonious place for all of them doesn't mean, as a team their response will be same and how they deal with situation.

Reply:
Saying someone is showing double standards would constitute in [Ad Hominem](https://en.wikipedia.org/wiki/Ad_hominem) rather than an actual argument.

All decisions must always be taken wrt the current scenario. Disregarding someone's opinion because of past behavior is wrong.

Also, people change. Thus I think the "double standards" is a wrong term here. Having double standards at a single point of time clearly indicates lack of moral clarity. Having a change of standards with time is natural and expected.

Reply:
Yeah true, this is a more fundamental statement than having "double standards". Will add "Ad hominem" thing instead.